kubernetes/update_images.py

```diff
@@ -2,15 +2,31 @@
 
 import json
 import subprocess
+from argparse import ArgumentParser
 
 import docker
 from docker.models.images import Image
 
+IMAGE_TYPES = ["cpp", "python"]
 REPO_NAME = "humancompatibleai/goattack"
 
 
 def main():
     """Main entry point."""
+    parser = ArgumentParser(
+        description="Builds and pushes new KataGo images to Docker Hub.",
+    )
+    parser.add_argument(
+        "--image",
+        type=str,
+        choices=IMAGE_TYPES,
+        default=IMAGE_TYPES,
+        help="Which images to update",
+        nargs="+",
+    )
```

Comment:
`match` only needs the `cpp` image, not the `python` image, so I'm adding a flag to `update_images` that allows specifying which images to build